websites/N/Night.design/presence.ts

```diff
@@ -0,0 +1,82 @@
+const presence = new Presence({
+	clientId: "1031730262092042321",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
+	};
+
+	presenceData.startTimestamp = Math.floor(Date.now() / 1000);
+	presenceData.buttons = [
+		{
+			label: "View Website",
+			url: "https://night.design/",
+		},
+	];
+
+	const domain = document.location.pathname.slice(3);
```

Comment:
Somewhat confusing variable name. `path` or `pathname` might be a better one.

Reply:
> Somewhat confusing variable name. `path` or `pathname` might be a better one.

I have change it to pathname